sci-libs/libspatialindex/libspatialindex-1.8.5.ebuild

```diff
@@ -0,0 +1,37 @@
+# Copyright 2014 Julian Ospald <hasufell@posteo.de>
+# Distributed under the terms of the GNU General Public License v2
+# $Header: $
+
+EAPI=5
+
+inherit autotools eutils
+
+MY_PN="spatialindex-src"
+MY_P="${MY_PN}-${PV}"
+
+DESCRIPTION="General framework for developing spatial indices"
+HOMEPAGE="http://libspatialindex.github.com/"
+SRC_URI="http://download.osgeo.org/libspatialindex/${MY_P}.tar.bz2"
+LICENSE="MIT"
+
+KEYWORDS="~amd64 ~x86"
+SLOT="0/4"
+IUSE="debug static-libs"
+
+S=${WORKDIR}/${MY_P}
+
+src_prepare() {
+	epatch "${FILESDIR}"/${PN}-1.8.1-QA.patch
+	eautoreconf
+}
+
+src_configure() {
+	econf \
+		$(use_enable static-libs static) \
+		$(use_enable debug)
+}
+
+src_install() {
```

Comment:
Well, I see this overlays as a place for user-maintained science-related ebuilds and ebuilds, which have QA issues. Anyhow, I will merge this request as soon as travis has verified it.